content/review/content-management.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Content Management"
+metaTitle: "sensenet - Content Management"
+metaDescription: "sensenet Content Management"
+---
+
+# What does content management mean?
+
+For content management we mean the processes of storing, editing, organizing and delivering content.
+# What is content regarding to sensenet?
+In sensenet everything is a content. Documents, tasks, users, projects, even system configuration files – everything – are stored and managed in the [content repository](/concepts/content-repository). It is the foundation that makes content management easy and powerful at the same time. All content items are organized in a [content tree](/concepts/content-tree), so you can always think in subtrees and inheritance when managing and presenting content.
+
+# Basic content management actions in sensenet
+
+All the basic content management actions are available in sensenet, such as:
+- create
+- edit (update)
+- delete
+- copy
+- move
+- preview
+- share
+- search
+- upload / download
+- organize (structure)
+
+To learn more go to [Actions](/concepts/actions)
+
+# Can anybody do anything?
+
+Content management is a collaborative process and every user has his own part in it. At this point we can talk about roles and permissions. [role based permission, document level permission], so basically what the user can or cannot do with the certain content.
+There is a set of basic built-in permissions, such as See, Open, Save, Delete etc., and it is even possible to add custom ones as well.
+The tree based hierarchical content structure in sensenet ensures permission inheritance, so it is enough to set a permission rule one time and it comes down in lower levels of the tree. This means, if you set a permission on a document library it will be automatically applied to subfolders and contents.
```

Comment:
Since sensenet has a tree based hierarchical content structure it ensures permission inheritance, so it is enough to set a permission rule one time and it comes down in lower levels of the tree. This means, if you set a permission on a document library it will be automatically applied to subfolders and contents.

Reply:
még kerekíteni rajt egy picit